docs/core/testing/unit-testing-mstest-runner-intro.md

```diff
@@ -0,0 +1,253 @@
+>   file extension:
+>     package: 'Contoso.MyTests', version: '1.0.0'
+>     path: 'Contoso.MyTests.dll'
+>     previously found assembly: 'S:\t\Contoso.MyTests\bin\Debug\net8.0\Contoso.MyTests.exe'
+> ```
+
+#### Using `dotnet test`
+
+> > [!IMPORTANT]
+> `dotnet test` is currently working only with MSTest, NUnit and xUnit tests. TPTF tests will not run using `dotnet test` and no error will be reported.
+
+For tests authored with MSTest, NUnit or xUnit test framework, it is possible to run tests with `dotnet test`.
+
+Provide a path to the tested dll, or to the project and your tests will run.
+
+### In Visual Studio
+
+Testing.Platform tests can be run (and debugged) in VisualStudio, they integrate with Test Explorer, and can also be run directly as Startup Project.
+
+#### Running the application directly in VS
+
+Testing.Platform test project are built as executables, and can be run directly. This will run all the tests in the given executable, unless a filter is provided.
+
+1. Navigate the test project you want to run in Solution Explorer, right click it and select `Set as Startup Project`.
+
+1. Go to Debug > Start without Debugging, (or press `Ctrl + F5`) to run the selected test project.
+
+Console window will pop up with the execution and summary of your test run.
+
+#### Debugging the application directly in VS
+
+Testing.Platform test project are built as executables, and can be debugged directly. This will debug all the tests in the given executable, unless a filter is provided.
+
+1. Navigate the test project you want to run in Solution Explorer, right click it and select `Set as Startup Project`.
+
+1. Set breakpoint into the test that you'd like to debug.
+
+1. Go to Debug > Start Debugging, (or press `F5`) to debug the selected test project.
+
+All tests will be executed until your test with a breakpoint is reached. Step through your test to debug it. Once you are done debugging the application will resume running all remaining tests, unless you stop it.
+
+### Using Test Explorer
+
+Testing.Platform tests partially integrate with Test Explorer. Tests can be Run and Debugged from Test Explorer.
+
+> [!IMPORTANT]
+> Automatic update of tests without building the project is not yet available.
+
+To run a test, navigate to Test Explorer, select the test (or tests) to run. Right click it, and choose `Run`.
+
+Similarly to debug a test, select the test (or tests) in Test Explorer, right click and choose `Debug`.
+
+## In Continuous Integration (CI)
+
+There is no special pipeline task, or any additional tooling to run Testing.Platform tests. There is also no additional tooling to run multiple tests projects through a single command.
+
+To run a test project in CI add one step for each test executable that you wish to run, such as this on AzureDevOps:
+
+```yml
+- task: CmdLine@2
+  displayName: "Run Contonso.MyTests.exe"
+  inputs:
+    script: '.\Contoso.MyTests\bin\Debug\net8.0\Contoso.MyTests.exe'
+```
+
+## Test Reports
```

Comment:
I would group this and the following entries under a `## Extensions` section

Reply:
You probably meant also the Troubleshooting part, but I like that it is separate, but see for yourself and do as you please :)